Lib/test/test_inspect/test_inspect.py

```diff
@@ -1872,11 +1926,14 @@ def static_method(): pass
             def a_property(self): pass
         class Slotermeyer:
             __slots__ = 'a_slot',
+        attribute_slot = Slotermeyer.__dict__['a_slot']
+        frame_locals = types.FrameType.f_locals
```

Comment:
I still think those should be closer to the test they are used in.

Reply:
I don't understand. `attribute_slot` is a variable, so you can move that variable where it's being used (line 1976), same for `frame_locals`. Don't move existing code, only move newest code.

Reply:
The problem with consistency, as perceived by me, is that if I move `attribute_slot` and `frame_locals`, I should also move e.g. `a_lambda`, and probably also the local `def`/`class` statements... But I find it a minor one, so I have no problem with adjusting the code according to your demand.

Reply:
Hm... I propose, however, to get rid of the entire `a_lambda` variable – it's redundant, as the code without it does not become less readable (rather quite contrary).

Reply:
You are improving the tests. While consistency is important, readability is usually more for the tests. You declare a variable that is only used once and quite far from it. It's better to 1) not declare the variable 2) or declare it and use it just after. In general, we don't touch old code if there is no need to (e.g., there is no issue with it) but we can improve new code. I'm not sure I'll have time to review your changes today and won't be there for the next 10 days.

Reply:
@picnixz For the record, according to your suggestion, the assignments to `attribute_slot` and `frame_locals` as well as (for consistency) some other stuff – have been moved, so that now they are as close as possible to the places when they are actually used.